app/src/main/java/org/thoughtcrime/securesms/components/menu/ActionItem.kt

```diff
@@ -13,5 +13,5 @@ data class ActionItem(
   val action: Runnable,
   val contentDescription: Int? = null,
   val subtitle: ((Context) -> CharSequence?)? = null,
-  @ColorRes val color: Int? = null,
+  @ColorInt val color: Int? = null,
```

Comment:
Only usage is passed in as `ColorRes`, did you want to change that?

```
ConversationReactionOverlay.kt

// Delete message
        if (userCanDeleteSelectedItems(context, message, openGroup, userPublicKey, blindedPublicKey)) {
            items += ActionItem(R.attr.menu_trash_icon, R.string.delete, { handleActionItemClicked(Action.DELETE) }, R.string.AccessibilityId_delete_message, message.subtitle, R.color.destructive)
        }
```

Reply:
Yep, that is now using the proper access for red, which is a new color attribute called `danger`, since different themes have different reds